### DISCUSSION TITLE
教程：如何本地部署文档

### DISCUSSION THREAD

**Discussion by sxzz** [element-plus/element-plus#7230] [Documentation]
点击 [此处](https://github.com/element-plus/element-plus/archive/refs/heads/gh-pages.zip) 下载最新文档压缩包。

文档不能运行在 `file://` 协议下，所以需要启动 HTTP 服务器运行。您可以使用 [serve](https://www.npmjs.com/package/serve) 来快速启动一个服务器，或使用 nginx。（这里不做介绍）

```bash
npm i -g serve
cd /some/path # 切换到压缩包解压后的路径
serve
```

**Comment by karstum** [comment]
请问一下, 文档下下来, 源文件类似被打包过的. 想要二次编辑文档作为二次封装UI后的新文档, 该如何做呢?

**Reply by KimYangOfCat** [reply]
你可以试试下面的操作步骤：
1. 下载 release 中对应版本的源码，
2. 执行 nvm install 安装对应版本的nodejs
3. 执行 pnpm install 安装依赖，
4. 编辑 docs 中内容，
5. 然后在项目根目录下执行  pnpm run docs:build 编译文档

**Reply by Wings-Fantasy** [reply]
我本地部署成功了，我要从哪把翻译文件也放到项目里？

**Comment by BrysonLin** [comment]
> 点击 [此处](https://github.com/element-plus/element-plus/archive/refs/heads/gh-pages.zip) 下载最新文档压缩包。
> 
> 文档不能运行在 `file://` 协议下，所以需要启动 HTTP 服务器运行。您可以使用 [serve](https://www.npmjs.com/package/serve) 来快速启动一个服务器，或使用 nginx。（这里不做介绍）
> 
> ```shell
> npm i -g serve
> cd /some/path # 切换到压缩包解压后的路径
> serve
> ```请问你这个包，是怎样生成的？能教一下嘛？我只会英文。我想本地启动个中文版本的。谢谢
